modules/gateway/variables.tf

```diff
@@ -56,9 +56,20 @@ variable "gateway_enable_private_access" {
 }
 
 variable "disable_agw" {
-  description = "Disables the creation of application gateway from the terraform module."
+  description = "Disables the creation of Application Gateway by the terraform module."
   type        = bool
-  default     = false
+}
+
+variable "context_path" {
+  description = "The context path for the external Application Gateway. The context path must start with '/' and must not end with '/'."
+  type        = string
+
+  validation {
+    condition = (
```

Comment:
why not transform to the proper syntax if the leading / is missing and remove the trailing / if present?